Formula/harmony-one.rb

```diff
@@ -0,0 +1,79 @@
+class HarmonyOne < Formula
+  desc "CLI tool to interact with the Harmony Blockchain"
+  homepage "https://docs.harmony.one/"
+  url "https://github.com/harmony-one/harmony/archive/refs/tags/v4.2.1.tar.gz"
+  sha256 "d29609a99a3a2e9031d2a26414ffdc3e271c3bf9cf9331dfcb321715fef682d4"
+  license "LGPL-3.0-or-later"
+
+  depends_on "go@1.16" => :build
+
+  depends_on arch: :x86_64
+  depends_on "gmp"
+  depends_on "openssl@1.1"
+
+  on_linux do
+    depends_on "gcc"
+    depends_on "glibc"
```

Comment:
Why? Does this software really need latest glibc?

Reply:
The software doesn't need the latest. 

I just figured out that Homebrew for Linux required those dependencies too, I had removed it from my local repo, but haven't hard push yet. 

I am waiting for other fixes, then hard push at once. 😄